### PR TITLE
COMMERCE-12589 CSRF bypass related to `redirect` in Commerce Catalogs - Fix functional tests

### PR DESCRIPTION
**Pull request by saramart81**
sent from #4398 
https://liferay.atlassian.net/browse/COMMERCE-12589

### CODE REVIEW

**Comment by saramart81** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceShipments.testcase [hunk 1]
```diff
@@ -2028,7 +2030,9 @@ definition {
 		}
 
 		task ("Then from the Shipment portlet the tracking URL is present as Base Tracking URL") {
-			Navigator.gotoNavTab(navTab = "Shipments");
+			CommerceNavigator.gotoPortlet(
```
Comment (baharturk) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceShipments.testcase:2033]: Can you use this one instead?
CommerceShipments.openShipmentsAdmin();

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceShipments.testcase [hunk 2]
```diff
@@ -691,7 +691,9 @@ definition {
 		}
 
 		task ("And the shipping method and tracking URL are correct") {
-			Navigator.gotoNavTab(navTab = "Shipments");
+			CommerceNavigator.gotoPortlet(
```
Comment (baharturk) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceShipments.testcase:694]: Can you use this one instead?
CommerceShipments.openShipmentsAdmin();

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceShipments.testcase [hunk 3]
```diff
@@ -2123,27 +2121,31 @@ definition {
 			productSkuList = "MIN55857,MIN55858,MIN55859",
 			warehouseName = "Italy");
 
-		Click(locator1 = "Icon#BACK");
+		CommerceNavigator.gotoPortlet(
```
Comment (baharturk) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceShipments.testcase:2124]: Can you use this one instead?
CommerceOrders.openOrdersAdmin();

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceShipments.testcase [hunk 4]
```diff
@@ -2123,27 +2121,31 @@ definition {
...
-			locator1 = "CommerceOrders#ORDER_ELLIPSIS_BY_STATUS");
-
-		CommerceEntry.clickNoError(menuItem = "View");
+		CommerceNavigator.gotoOrderEntryViaAdmin(status = "Partially Shipped");
 
 		CommerceOrders.createShipment(
 			productSkuList = "MIN55860,MIN55861",
 			warehouseName = "Italy");
 
+		CommerceNavigator.gotoPortlet(
```
Comment (baharturk) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceShipments.testcase:2138]: Can you use this one instead?
CommerceOrders.openOrdersAdmin();

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceShipments.testcase [hunk 5]
```diff
@@ -2686,16 +2686,22 @@ definition {
 				productSkuList = "MIN55861",
 				warehouseName = "Italy");
 
-			CommerceEntry.gotoMenuTab(menuTab = "Shipments");
-
-			Click(
-				key_rowNumber = 1,
-				locator1 = "CommerceNavigation#ENTRY_NAME_N");
+			CommerceNavigator.gotoPortlet(
```
Comment (baharturk) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceShipments.testcase:2689]: Can you use this one instead?
CommerceShipments.openShipmentsAdmin();

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceOrders.testcase [hunk 6]
```diff
@@ -2576,7 +2578,9 @@ definition {
 		}
 
 		task ("Then the status of the order is updated as Shipped") {
-			Click(locator1 = "Icon#BACK");
+			CommerceNavigator.gotoPortlet(
```
Comment (baharturk) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceOrders.testcase:2581]: Can you use this one instead?
CommerceOrders.openOrdersAdmin();

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceOrders.testcase [hunk 7]
```diff
@@ -2558,7 +2558,9 @@ definition {
 		}
 
 		task ("Then the status of the order is Partially Shipped") {
-			Click(locator1 = "Icon#BACK");
+			CommerceNavigator.gotoPortlet(
```
Comment (baharturk) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceOrders.testcase:2561]: Can you use this one instead?
CommerceOrders.openOrdersAdmin();

**Review by baharturk — CHANGES_REQUESTED** [review]
Please take a look at my comments, thanks!

**Comment by saramart81** [comment]
ci:test:sf

**Review by baharturk — APPROVED** [review]

**Comment by saramart81** [comment]
ci:test:relevant

**Comment by saramart81** [comment]
ci:test:functional

**Comment by saramart81** [comment]
ci:test:commerce-functional

**Comment by saramart81** [comment]
ci:test:sf

**Review by baharturk — APPROVED** [review]

**Comment by saramart81** [comment]
ci:test:relevant

**Comment by saramart81** [comment]
ci:test:commerce:functional

**Comment by saramart81** [comment]
ci:forward

**Comment by saramart81** [comment]
ci:test:relevant

**Comment by saramart81** [comment]
Hi  @victoroscuro this PR is good please help force-forward it, thanks! @victoroscuro

**Comment by victoroscuro** [comment]
ci:forward:force

**Pull request closed without merging**